scripts/notarize-macos-dmg.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright (C) 2024-2024	The DOSBox Staging Team
+
```

Comment:
Maybe mention that the script must be run on macOS and no there are no extra prerequisites (all commands used by the script are included in the stock OS).

Reply:
I'll mention in the description comment below that Xcode is required